app/controllers/api/admin/news_items_controller.rb

```diff
@@ -8,13 +8,13 @@ def index
       end
 
       def show
-        news_item = NewsItem.with_pk!(params[:id])
+        news_item = ::News::Item.with_pk!(params[:id])
```

Comment:
Should this technically mean a rename of the controller to follow standard patterns?

Reply:
```ruby
news/items_controller.rb
```

Reply:
decided to leave it since we're probably not adding more news controllers